File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char,int> mpp;
        int count=0;
        for(auto x : s){
            mpp[x]++;
            count++;
        }
        for(auto x : t){
            if(mpp[x] == 0){
                return false;
            }
            mpp[x]--;
            count--;
        }
        if(!count){return true;}
        else{
            return false;
        }
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mpp = {}
        for i in nums:
            if mpp.get(i) == None:
                mpp[i] = 0
            mpp[i] += 1
        lst = []
        for key,value in mpp.items():
            lst.append([-value, key])
        lst.sort()
        res = []
        for i in range (0,k):
            res.append(lst[i][1])
        return res
        